src/bridge/cockpitchannel.c

```diff
@@ -977,6 +978,7 @@ cockpit_channel_parse_connectable (CockpitChannel *self,
   const gchar *internal;
   const gchar *address;
   JsonObject *options;
+  gboolean local = FALSE;
```

Comment:
Shouldn't we default to TRUE and only set false when we have the address argument?